clojure-mode.el

```diff
@@ -352,6 +352,13 @@ Called by `imenu--generic-function'."
               (set-match-data (list def-beg def-end)))))
         (goto-char start)))))
 
+(eval-when-compile
+  ;; See clojure.lang.LispReader definition and getMacro invocation(s)
```

Comment:
If we're going to link I think we should link to a line in a specific revision, e.g. the tag for clojure 1.7.  If just put a link to a line number in a file on master we might get in a situation where there's something else on that line or that the file itself has been moved/deleted.